protocols.go

```diff
@@ -1,7 +1,7 @@
 package diag
 
 import (
-	"github.com/florianl/go-diag/internal/unix"
+	"github.com/scitags/go-diag/internal/unix"
```

Comment:
Looks like an left-over:
```suggestion
	"github.com/florianl/go-diag/internal/unix"
```

Reply:
My bad! I just rebased to avoid having a commit devoted to the fix. Sorry about that!